### ISSUE TITLE
Chrome shows authentication dialog even if correct credentials are provided

### ISSUE THREAD

**Issue by younesaassila** [younesaassila/ttv-lol-pro#160]
**Workarounds:**

- Clicking on "Cancel" closes the dialog and requests may be proxied.
- Save proxy credentials in the browser's password manager (popup will still appear but credentials will be autofilled and clicking "Sign in" works).

**Comment by younesaassila** [comment]
Should also directly provide creds in Firefox in proxyInfo

**Comment by younesaassila** [comment]
Seems like requests made as soon as the page is starting to load (like www.twitch.tv or passport.twitch.tv) don't trigger the event handler, thus no credentials are provided automatically. I've got to assume this is a Chrome bug at this point

**Comment by younesaassila** [comment]
Also even with Laissez-passer turned on Chrome just refuses to proxy www.twitch.tv?? And will then show the auth dialog for passport.twitch.tv because the event handler isn't getting called. UGHH

**Comment by mariusvn** [comment]
same here, i use an https proxy

**Comment by jkroepke** [comment]
> * Save proxy credentials in the browser's password manager

Does not work. Chrome is asking. I used https://chromewebstore.google.com/detail/proxy-auto-auth/ggmdpepbjljkkkdaklfihhngmmgmpggp now

while it not works on initial browser start (because chrome load extensions after start), it works well if I navigate to twitch

**Comment by younesaassila** [comment]
@jkroepke It's asking but at least you can just click Continue and it proxies correctly

**Comment by jkroepke** [comment]
> @jkroepke It's asking but at least you can just click Continue and it proxies correctly

Thats true, but the popup it still coming. The Extension eleminates this

**Comment by younesaassila** [comment]
It's been months since anyone reported having this issue, perhaps the issue has been fixed in a Chromium browser update. I'll close this issue.

**Comment by younesaassila** [comment]
A fix for unproxied PlaybackAccessToken requests in 2.6.0 brings back this issue. This issue is 100% caused by Chromium not calling the onAuthRequired handler in basically all extension contexts apart from service workers. I tried to fetch a resource using proxies needing authentication from the background worker, which works, but doesn't fix the issue for requests coming from Twitch.

Multiple reports:
- https://github.com/foxyproxy/browser-extension/issues/145
- https://issues.chromium.org/issues/40880379
